tests/trace/test_evaluate.py

```diff
@@ -324,3 +324,38 @@ def score(output):
         "model_latency": {"mean": pytest.approx(1, abs=1)},
     }
     assert time.time() - now < 5
+
+
+def test_evaluation_from_weaveobject_missing_evaluation_name(client):
+    dataset_rows = [{"input": "1 + 2", "target": 3}, {"input": "2**4", "target": 15}]
```

Comment:
dataset_rows is also defined on line 9, any reason to duplicate?

Reply:
Yes, for some reason it didn't like using the one outside.  I think it's better practice to scope these per test anyways.  We can convert to fixtures in another PR :)